servicetalk-http-api/src/main/java/io/servicetalk/http/api/BaseHttpBuilder.java

```diff
@@ -73,14 +72,25 @@
 
     /**
      * Enables wire-logging for connections created by this builder.
-     * <p>
-     * All wire events will be logged at {@link Level#TRACE TRACE} level.
```

Comment:
Comment for all occurances of `enableWireLogging` method:

Should we mark the existing `enableWireLogging` methods as deprecated? If not and you prefer to keep both, it's better to keep this info in javadoc, add desciption that this method logs all user data, and reference another overload if users need more customization.

Reply:
I'll just deprecate the methods and remove the logs for now, we can remove them in the next release.